packages/next-tinacms-cloudinary/src/cloudinary-media-store.ts

```diff
@@ -16,7 +16,7 @@ export class CloudinaryMediaStore implements MediaStore {
   fetchFunction = (input: RequestInfo, init?: RequestInit) => {
     return fetch(input, init)
   }
-  accept = 'text/*,application/*,image/*'
+  accept = 'text/*,application/pdf,image/*'
```

Comment:
did you intend to restrict accept from application/* to application/pdf (ie was it incorrect before?)

Reply:
Maybe we could make accept configurable @jamespohalloran ?

Reply:
> did you intend to restrict accept from application/* to application/pdf (ie was it incorrect before?)

@kldavis4 I'm not sure if this is a bug with react-dropzone, but unless I specified application/pdf specifically, PDF's weren't selectable with the file-browser. I can poke around and see if there's a workaround

<img width="660" alt="Screen Shot 2023-03-01 at 4 10 22 PM" src="https://user-images.githubusercontent.com/3323181/222254655-a6215ed2-949b-44d5-a387-8e4435a3b47b.png">

> Maybe we could make accept configurable @jamespohalloran ?

Yeah that might be nice. I think we'd probably want to do it on a per-field level though, rather than here at the media provider level

Reply:
@kldavis4 I hardcoded a list of some common potential `application/` mime-types for now, since `application/*` doesn't seem to be supported.

@logan-anderson I was thinking maybe we could make this configurable in a future PR? I think if this is configurable on a per-field basis, it might not be super trivial